### ISSUE TITLE
Linux FF 3.5 with Feed Sidebar 4.0 - New RSS Articles are not appearing

### ISSUE THREAD

**Issue by cfinke** [cfinke/Feed-Sidebar#26]
<b>What steps will reproduce the problem?</b>
1. Have several livefeed bookmarks in your list
2. install feed sidebar in Firefox 3.5
3. Click 'update/refresh' Wait several hours. Click 'update/refresh'.

<b>What is the expected output? What do you see instead?</b>

Expect to see new rss feed posts.  Out of 31 feeds I have, only TWO are
showing new articles, and not all of those either.  If I examine the
livefeed myself, it shows many new articles present.

<b>What version of the product are you using? On what operating system?</b>

OpenSuSE Linux v11.0, updated to FireFox 3.5 on July 1, 2009, installed
Feed Sidebar v4.0 from Mozilla Addons. Was using feed sidebar previously on
version FF3.0.11 successfully. (previous version of Sidebar though)

<b>Please provide any additional information below.</b>

Not sure what's changed really.  Just not able to see new rss articles
using feed sidebar.  Actually went to the trouble to try the other rss feed
'things' out there... ick!  Hoping this is a simple typo or oops concerning
the 'seen/notseen' status or something similar.

As given above, am a seasoned user with your addon, very much enjoy it, no
problems or complaints until new v4.0 for FF3.5. {Sigh}

---

Issue imported from Google Code
Author: [lornix13](http://code.google.com/u/lornix13/)
Published: 2009-07-03T09:26:25.000Z
Status: Done
Link: http://code.google.com/p/feedsidebar/issues/detail?id=27

**Comment by cfinke** [comment]
Was able to get it to do something different.  Still not working as expected, but
it's not doing the same thing now.

Turned off auto-updates.

Settings are now:  No auto-updates.  Show ALL articles.  Hide read articles. (open in
new-tabs, close when mark all read.. etc)

NOW, when I click on the 'update/refresh' button, I get a yellow area with "x: Feed
has no content" for EVERY feed.  (wee!  32 boxes!)

I'll keep tinkering.

---

Comment imported from Google Code
Author: [lornix13](http://code.google.com/u/lornix13/)
Published: 2009-07-03T09:33:16.000Z
Link: http://code.google.com/p/feedsidebar/issues/detail?id=27#c1

**Comment by cfinke** [comment]
Fixed it!!

Uninstalled Feed Sidebar, then grep'd for all occurrances of options and whatnot in
firefox subdir.  I suppose that's not for the faint of heart, but it certainly
worked.  When I restarted firefox and reinstalled Feed Sidebar, all the articles came
up as expected.  Hooray!   

Thank you very much for this addon.

---

Comment imported from Google Code
Author: [lornix13](http://code.google.com/u/lornix13/)
Published: 2009-07-03T09:53:14.000Z
Link: http://code.google.com/p/feedsidebar/issues/detail?id=27#c2
